packages/mobile/src/components/feed-tip-tile/SendTipButton.tsx

```diff
@@ -19,6 +19,7 @@ const useStyles = makeStyles(({ spacing, palette, typography }) => ({
     marginTop: spacing(4)
   },
   sendTipButtonTitleContainer: {
+    paddingTop: spacing(0.5),
```

Comment:
It's hard to see while reviewing, but curious why this fixes the vertical centering? Looks like the flex should have already centered it - what was throwing it off?

Reply:
good callout... i dont like this change... but flex didnt center the content perfectly. i added a background color on the icon and the text to see if they were the culprit but they didn't seem to be. somehow the centering is just a bit off (by about 2px)